Functions_Collections_Homework4.py

```diff
@@ -0,0 +1,42 @@
+from string import ascii_lowercase  # we will use it to generate keys as letters
+from random import randint, choice  # we will use it to generate random integers for dictionary values and list length; choice will be used to select random letter for keys
+
+
+# create new dicts where key = key from random list and value = list of values for this key
+def values_to_list(my_list):
```

Comment:
It would be better to use "docstrings" if you want to describe what function should do. You just need to put text in triple quotes after the function declaration, for example:
```
def values_to_list(my_list):
    """create new dicts where key = key from random list and value = list of values for this key"""
```
This makes your code better documented, and you can simply use __doc__ to get function description:
```
print(values_to_list.__doc__)
```
The output: 
![image](https://user-images.githubusercontent.com/34323554/107676013-59f9a700-6ca1-11eb-8bb4-34c7fcd0d12f.png)

More details: https://www.programiz.com/python-programming/docstrings

Reply:
Thank you, will use that! I was using comments as in hometask it's said "comment every line of your code".